contrib/ci-workers/templates/buildbot-worker.service.j2

```diff
@@ -7,9 +7,9 @@ After=network.target
 Type=forking
 PIDFile=/home/{{ buildbot_worker_user }}/{{ buildbot_worker_name }}/twistd.pid
 WorkingDirectory=/home/{{ buildbot_worker_user }}
-ExecStart=/usr/local/bin/buildbot-worker start {{ buildbot_worker_name }}
-ExecReload=/usr/local/bin/buildbot-worker restart {{ buildbot_worker_name }}
-ExecStop=/usr/local/bin/buildbot-worker stop {{ buildbot_worker_name }}
+ExecStart={{ pip_bin_dir }}/buildbot-worker start {{ buildbot_worker_name }}
```

Comment:
Why was the directory changed?

Reply:
It isn't: `pip_bin_dir` is set to the same location by default. This parametrisation was necessary because in some environments, pip installs executables in a different location.